### ISSUE TITLE
Scroll not working on mobile devices, only desktop

### ISSUE THREAD

**Issue by AyoParadis** [rafgraph/react-router-hash-link#97]
The scroll only works if you double tab. single tap scrolls to the top of the screen on mobile across all browsers. 

https://css-tricks.com/annoying-mobile-double-tap-link-issue/ this solution did not work

**Comment by CaliforniaLuv** [comment]
That's right, I also found the bug after the belated release, so I used the damn jquery.

**Comment by rafgraph** [comment]
Scrolling on mobile works fine in the example app on latest iOS and Android https://react-router-hash-link.rafgraph.dev/ so my guess is it’s an edge case issue (or an issue with your code).

Please provide a failing example using code sandbox with steps required to reproduce. Thanks.
